etc/app_cov.mk

```diff
@@ -0,0 +1,88 @@
+CUSTOM_EXEC=1
```

Comment:
`# Include this file into an application/target/cov/Makefile to build and run coverage for the app.`

Reply:
Added.